data/seeds/01_users.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('users').truncate()
+    .then(function () {
+      // Inserts seed entries
+      return knex('users').insert([
+        {username: 'Crash', password: 'UkaUka', registered: true},
+        {username: 'Groot', password: 'IamGroot', registered: true},
```

Comment:
I suggest that you do not seed user accounts into the database because security is compromised since there is no hashing occurring.